source/feature-manager.tsx

```diff
@@ -47,8 +47,11 @@ type FeatureLoader = {
 } & Partial<InternalRunConfig>;
 
 type InternalRunConfig = {
+	/** Every condition must be true */
 	asLongAs: BooleanFunction[] | undefined;
+	/** At least one condition must be true */
 	include: BooleanFunction[] | undefined;
+	/** No conditions must be true */
 	exclude: BooleanFunction[] | undefined;
```

Comment:
Probably more readable as "Every condition must be false", both here and in the place you copied this from